tests/benchdnn/inputs/binary/test_binary_gpu

```diff
@@ -87,6 +91,10 @@
 --sdt=f32:f32
 --batch=option_set_all
 
+--ddt=s32
+--sdt=s32:s32
+--batch=option_set_all
```

Comment:
Do you need to remove the unimplemented statement here? 

https://github.com/yehudaorel/oneDNN/blob/gpu_binary_s32/tests/benchdnn/binary/binary.cpp#L150-L156